### ISSUE TITLE
Consistent focus ring color

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-style#290]
Focusable elements should have a consistent `:focus` contrasting color and treatment clearly distinguished from primary color palette.
